src/traccuracy/track_errors/divisions.py

```diff
@@ -6,7 +6,7 @@
 from collections import Counter
 from typing import TYPE_CHECKING
 
-from traccuracy._tracking_graph import NodeAttr
+from traccuracy._tracking_graph import NodeFlag
```

Comment:
This comment applies broadly to this file. If we want to enforce the pattern of avoiding ragged attributes on our graphs, then we need to add a few lines here to set all flags to a default and then flip them as needed. This would match the pattern in the ctc functions.

Thoughts on including this change in this PR vs deferring to a separate one?

Reply:
@msschwartz21 perhaps should be in a separate PR? I'm wondering where you think it's best to first set them?

We could set them on graph creation, such that they are available and default `False` until the metrics are computed.

We could set them when the user requests a metrics computation, that way their mere presence indicates the graph has had metrics computed?

I kind of like having them on the graph from the beginning, but then there's nothing that can quickly tell you whether or not a graph has had its errors annotated...

Reply:
Given the need for discussion, let's leave this for another PR.

I don't want to put these flags on the graph at initialization because it will create more interdependency between the loaders and the errors. Right now someone adding a new metric/error shouldn't need to touch the loaders or `TrackingGraph`.

I think it's easier to slot them in at the beginning of metric calculation. We also added flags to `TrackingGraph` that record if node/edge/division errors have been annotated.

Reply:
I opened #129 for this discussion, and will mark this as resolved for this PR.